src/enzo/phys_constants.h

```diff
@@ -70,6 +70,16 @@
 
 #define SolarMass                       1.9891e33
 
+/* Solar Radius [cm] */
+
+#define SolarRadius                     6.96e10
+
+/* Solar Luminosity [erg/s] */
+
+#define SolarLuminosity                 3.9e33
```

Comment:
I see that SolarLuminosity is only used in a print statement conversion right now, but it would be good to change the definition of SolarLuminosity to 3.839e33 for higher accuracy in future use cases.

Reply:
On this same topic, I've seen a few hard coded physical constants in the new code. There was a recent PR (in [bitbucket](https://bitbucket.org/enzo/enzo-dev/pull-requests/417)) that made all of these changes.

I would be fine with approving the changes without this modification and making an issue to address this later because APs would still function but be less consistent with the precision issues.

Reply:
I'm fine with that! I made similar comments throughout, but I understand that these are minor issues that can be addressed later (if they haven't already been addressed in the bitbucket PR).

Reply:
I think this comment is outside the scope of this PR. Someone should probably update this value in future though.